app/controllers/catalog_controller.rb

```diff
@@ -68,24 +67,96 @@ def self.modified_field
+
+    config.add_index_field solr_name('contributor', :stored_searchable),
+                           label: 'Contributor',
+                           itemprop: 'contributor',
+                           link_to_search: solr_name('contributor', :facetable)
+
+    config.add_index_field solr_name('proxy_depositor', :symbol),
+                           label: 'Depositor',
+                           helper_method: :link_to_profile
+
+    config.add_index_field solr_name('depositor'),
+                           label: 'Owner',
+                           helper_method: :link_to_profile
+
+    config.add_index_field solr_name('publisher', :stored_searchable),
+                           label: 'Publisher',
+                           itemprop: 'publisher',
+                           link_to_search: solr_name('publisher', :facetable)
+
+    config.add_index_field solr_name('based_near', :stored_searchable),
+                           label: 'Location',
+                           itemprop: 'contentLocation',
+                           link_to_search: solr_name('based_near', :facetable)
+
+    config.add_index_field solr_name('language', :stored_searchable),
+                           label: 'Language',
+                           itemprop: 'inLanguage',
+                           link_to_search: solr_name('language', :facetable)
+
+    config.add_index_field solr_name('date_uploaded', :stored_sortable, type: :date),
+                           label: 'Date Uploaded',
+                           itemprop: 'datePublished',
+                           helper_method: :human_readable_date
+
+    config.add_index_field solr_name('date_modified', :stored_sortable, type: :date),
+                           label: 'Date Modified',
+                           itemprop: 'dateModified',
+                           helper_method: :human_readable_date
+
+    config.add_index_field solr_name('date_created', :stored_searchable),
+                           label: 'Date Created',
+                           itemprop: 'dateCreated'
+
+    config.add_index_field solr_name('rights', :stored_searchable),
+                           label: 'Rights',
+                           helper_method: :license_links
+
+    config.add_index_field solr_name('resource_type', :stored_searchable),
+                           label: 'Resource Type',
+                           link_to_search: solr_name('resource_type', :facetable)
+
+    config.add_index_field solr_name('file_format', :stored_searchable),
+                           label: 'File Format',
+                           link_to_search: solr_name('file_format', :facetable)
+
+    config.add_index_field solr_name('identifier', :stored_searchable),
+                           label: 'Identifier',
+                           helper_method: :index_field_link,
+                           field_name: 'identifier'
+
+    config.add_index_field solr_name('embargo_release_date', :stored_sortable, type: :date),
```

Comment:
Most of these diffs are for the linelength cop. But sufia 7.2 did add quite a few new index fields. So added them as well.